cmd/e2e_test.go

```diff
@@ -138,6 +148,7 @@ var _ = Describe("end to end", func() {
 		testNew("second")
 		testBatch()
 		testLatest("0.1.0")
+		testNext("1.0.0")
```

Comment:
This looks good, will need to add another test like the one below on line 156 for the error case ( can not leave a comment closer as it is outside the scope of the changes ).